README.md

```diff
@@ -13,6 +13,8 @@ Notes for [developers]
 
 ## Test it out
 
+(For a fully comprehsensive getting started guide, see our [single node how-to](https://m3db.github.io/m3db/how_to/single_node/)).
```

Comment:
`https://m3db.github.io/m3db/how_to/single_node.html`

Reply:
The docs should work without the `index.html` suffix, I think it's cleaner.

https://m3db.github.io/m3db/architecture/ and https://m3db.github.io/m3db/architecture/index.html result in the same web page.